biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -534,6 +535,31 @@
 	String		LAUNCH_ACTIVATORS							= "launch.activators";
 	String		LAUNCH_ACTIVATION_EAGER						= "launch.activation.eager";
 
+	/**
+	 * Any attributes that should be removed from the attributes before
+	 * printing.
+	 */
+	String[]	INTERNAL_ATTRIBUTES							= {
```

Comment:
```suggestion
	Set<String>	INTERNAL_ATTRIBUTES							= Sets.of(
```

Can we make this `Sets.of`? This will catch duplicate entry mistakes and will improve lookup time during execution.

Reply:
I am not sure how it catches duplicates? It will silently ignore them? I also seriously doubt it is faster for these short lists and they add to initialization time ... but I've updated them.